mgr.go

```diff
@@ -653,6 +651,7 @@ func (mgr *SysboxMgr) prepMounts(id string, uid, gid uint32, shiftUids bool, pre
 			formatter.ContainerID{id})
 	}
 
+	info.mntPrepRev = []mntPrepRevInfo{}
```

Comment:
I think it would be cleaner to clear info.mntPrepRev in unregister(), right after it's used to revert the mount preparation actions.

Reply:
Done, I agree it's better to clean it up on unregister() since it's no longer needed afterwards.